app/furniture/journal/renderer.rb

```diff
@@ -6,7 +6,9 @@ def autolink(link, link_type)
     end
 
     def postprocess(doc)
-      doc.gsub(/@([a-zA-Z\d]*)@(.*\.[a-zA-Z]*)/, '<a href="https://\2/@\1">@\1@\2</a>')
+      doc
+        .gsub(/@([a-zA-Z\d]*)@(.*\.[a-zA-Z]*)/, '<a href="https://\2/@\1">@\1@\2</a>')
+      # .gsub(/#(\w+)/, '<a href="../terms/\1">#\1</a>')
```

Comment:
Leftovers; just wanted to commit it before I fell asleep.

Reply:
Should this commented-out code be removed, or are you keeping it for the future?

Reply:
It should not be in this branch; but the follow on one...